examples/mlperf/model_train.py

```diff
@@ -430,6 +430,7 @@ def train_bert():
   target, achieved                                      = getenv("TARGET", 0.72), False
 
   config["DEFAULT_FLOAT"] = dtypes.default_float.name
+  config["HALF_LINEAR"]   = getenv("HALF_LINEAR", 0)
```

Comment:
would be nice to also log dropout so it's easy to tell if dropout is used or not for now, not related to this pr though